SecurityPkg/Library/DxeImageVerificationLib/DxeImageVerificationLib.h

```diff
@@ -10,6 +10,9 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #ifndef __IMAGEVERIFICATIONLIB_H__
 #define __IMAGEVERIFICATIONLIB_H__
 
+#include <PiPei.h>
+#include <PiDxe.h>

```

Comment:
why we need those?

Reply:
HOST_APPLICATION should not include PiPei.h in general.

If this is for test only, I prefer to drop this change.